From 9890015baef3b353f143f4fddde7510eed89540d Mon Sep 17 00:00:00 2001
Message-Id: <9890015baef3b353f143f4fddde7510eed89540d.1228954349.git.christian@jaeger.mine.nu>
In-Reply-To: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
References: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
From: Christian Jaeger <christian@jaeger.mine.nu>
Date: Wed, 10 Dec 2008 23:07:07 +0100
Subject: [PATCH] Fix: quote parameters to get_device_opts

Or, when using set -eu, get_device_opts would stop with
 $2: unbound variable

Signed-off-by: Christian Jaeger <christian@jaeger.mine.nu>
---
 debian/initramfs/cryptroot-hook |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/debian/initramfs/cryptroot-hook b/debian/initramfs/cryptroot-hook
index 51ca537..815b133 100644
--- a/debian/initramfs/cryptroot-hook
+++ b/debian/initramfs/cryptroot-hook
@@ -348,7 +348,7 @@ add_device() {
 		fi
 
 		# Get crypttab root options
-		if ! get_device_opts $node $opts; then
+		if ! get_device_opts "$node" "$opts"; then
 			continue
 		fi
 		echo "$OPTIONS" >> "$DESTDIR/conf/conf.d/cryptroot"
-- 
1.6.0.5

